example/mm_jobman.sh

```diff
@@ -208,20 +212,30 @@ submit_each_line_with_mmfloat() {
         if [ "$dryrun" = true ]; then
             full_cmd+="#-------------\n"
         fi
-        cmd="$download_cmd$line$upload_cmd" 
-        full_cmd+="mmfloat submit -i '$image' -j <(echo -e '''$cmd''') -c '$c_value' -m '$m_value' $dataVolume_params\n"
- 
-    done < "$script_file"
 
-    # Remove the last '\n'
-    full_cmd=${full_cmd%\\n}
+        image_run=$(echo $line | awk '{print $3}')
+	subline+="mkdir -p /home/jovyan/TEMP/output && "
+        subline+=$(echo $line | awk '{ print substr($0, index($0,$4)) }')
```

Comment:
this is also no longer relvant because it is designed for `docker run` and we don't do that. Please revert